packages/core/utils/create-project.ts

```diff
@@ -56,7 +39,7 @@ export async function detectSvelteDirectory(directoryPath: string): Promise<stri
     return null;
 }
 
-export async function createProject(cwd: string) {
+export async function createProject(cwd: string, supportKit: boolean, supportSvelte: boolean) {
     const createNewProject = await booleanPrompt("Create new Project?", true);
     if (!createNewProject) {
         console.log("New project should not be created. Exiting.");
```

Comment:
this should be using `endPrompts` rather than `console.log` - it's also reiterating what's already said in the prompt, so we can just remove it and set it to just `Exiting.`

```suggestion
        endPrompts("Exiting.");
```